graphsdk/src/androidTest/java/com/microsoft/graph/http/UrlConnectionTests.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.graph.http;
+
+import android.test.AndroidTestCase;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test cases for {@see UrlConnection}
+ */
+public class UrlConnectionTests extends AndroidTestCase {
```

Comment:
Why would we need to test MockConnection - it is just a mock, no?  A class called UrlConnectionTests doesn't test the class it seems to be named after.

Also I don think it is interesting to test that this class can handle http connections, as it really depends on an underlying implementation to do that, however it has specific behavior around its construction and headers that would be good to validate. 

---
In reply to: [65908305](https://github.com/microsoftgraph/msgraph-sdk-android/pull/17#discussion_r65908305) [](ancestors = 65908305)